account_invoice_discount_formula/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Prefer png images.
Even better: you can add https://github.com/OCA/maintainer-tools/tree/master/template/module/readme files for letting the bot to generate this.